src/person/serializers.py

```diff
@@ -76,6 +77,25 @@ def get_current_match(self, user):
             return None
         return MatchSerializer(matches[0], user=user).data
 
+    def get_prompts(self, user):
+        prompt_questions = user.person.prompt_questions.all()
+        prompt_answers = user.person.prompt_answers
+        if prompt_answers is None:
+            return []
+        # We have to replace all single quotes with double quotes for JSON
+        prompt_answers = json.loads(prompt_answers.replace("'", '"'))
```

Comment:
This still bothers me and I think it's time to change it. For prompts, words with apostrophes like I'm and you're will be converted to I"m and you"re if I'm not mistaken.

https://stackoverflow.com/questions/1110153/what-is-the-most-efficient-way-to-store-a-list-in-the-django-models

From the 2nd answer, try
```
jsonDec = json.decoder.JSONDecoder()
prompt_answers = jsonDec.decode(prompt_answers)
```
The first answer looks like a custom class implementation which may come in handy for generalizing the CRUD methods so we don't need to waste time with copy+paste though I think it's overkill in your case. Let me know if this works and if so please find every instance in which you replace single quotes to double quotes and replace it.

(you may need to import simplejson instead of json)

Reply:
Fixed it, the solution was to explicitly use `json.dumps(['list goes here'])` when modifying the person model instead of letting Django convert the list to a json string automatically. I removed all of the `string.replace("'",'"')` calls.